CSS/core/variables.css

```diff
@@ -0,0 +1,4 @@
+:root{
+    --var-display: flex;
```

Comment:
do we need display type in variable, idea is to keep things in variable that will help us in theming later